templates/about/editorialPolicies.tpl

```diff
@@ -113,6 +117,40 @@
 </div>
 {/if}
 
+{if $currentJournal->getSetting('publishingMode') == $smarty.const.PUBLISHING_MODE_SUBSCRIPTION &&
+	$paymentConfigured &&
+	$currentJournal->getSetting('journalPaymentsEnabled') &&
+	$currentJournal->getSetting('acceptSubscriptionPayments') &&
+	$currentJournal->getSetting('purchaseIssueFeeEnabled') &&
+	$currentJournal->getSetting('purchaseIssueFee') > 0}
+<div id="purchaseIssue"><h3>{translate key="about.purchaseIssue"}</h3>
```

Comment:
Indent this line and the following ones, as they're inside the {if}, and move the <h3> onto its own line. This reads to me like an accidentally unterminated element.